src/Android/Xamarin.Android/Samples/Augmented Reality/CollectDataAR/readme.metadata.json

```diff
@@ -0,0 +1,42 @@
+{
+    "category": "Augmented reality",
+    "description": "Tap on real-world objects to collect data.",
+    "formal_name": "CollectDataAR",
+    "ignore": false,
+    "images": [
+        "CollectDataAR.png"
+    ],
+    "keywords": [
+        "attachment",
+        "augmented reality",
+        "capture",
+        "collection",
+        "collector",
+        "data",
+        "field",
+        "field worker",
+        "full-scale",
+        "mixed reality",
+        "survey",
+        "world-scale"
+    ],
+    "nuget_packages": {
+        "Esri.ArcGISRuntime.ARToolkit": "100.6.0",
+        "Esri.ArcGISRuntime": "100.6.0"
+    },
+    "offline_data": [],
+    "redirect_from": [
+        "/net/latest/android/sample-code/collectdataar.htm"
+    ],
+    "relevant_apis": [
+        "ARSceneView",
+        "GraphicsOverlay",
+        "SceneView",
+        "Surface"
+    ],
+    "snippets": [
+        "CollectDataAR.cs",
+        "MSLAdjustedARLocationDataSource.cs"
```

Comment:
Just noticed this is missing the snippet link for the AXML. Since this is using JoystickSeekBar, I also recommend adding references to Controls/JoyStickSeekbar, and Resources/values/Attrs.xml